src/Authentication.gs

```diff
@@ -1,3 +1,6 @@
+/* Circuit breaking constant*/
+var backendDissabled = true;
```

Comment:
How i thought about it right now, is not something that will triger automatically, its just for us, to use in a internal level. A simple function can toggle the var value, but i added those in case the back end is not ready in time. We can change it easily though.